src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchParam, setSearchParam] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => (
+    movie.title.toLowerCase()
+      .includes(searchParam.trim().toLowerCase())
+    || movie.description.toLowerCase()
```

Comment:
use extra variables to improve you code readable

Reply:
thanks, fixed